example/package_inproc.yml

```diff
@@ -203,8 +203,8 @@ graphs:
     - id: kwargs.predictions
       iotype: DIRECTORY
       source:
-        node_id: null
-        subgraph_id: infer_pipeline
+        node_id: infer_pipeline.sentiment
```

Comment:
node_id and subgraph_id swapped, I think it would be better idea to use package_inproc_argo.yml and not change this file as later we can compare the changes ?

Reply:
Fixed.